### PR TITLE
feat(hycu): add activate license modal

### PR DESCRIPTION
**Pull request by tibs245**
| Question         | Answer
| ---------------- | ---
| Branch?          | `feat-hycu`
| Bug fix?         | no
| New feature?     | yes
| Breaking change? | no
| Tickets          | Feat #MANAGER-15175
| License          | BSD 3-Clause

- [x] Try to keep pull requests small so they can be easily reviewed.
- [x] Commits are signed-off
- [x] Only FR translations have been updated
- [x] Branch is up-to-date with target branch
- [x] Lint has passed locally
- [x] Standalone app was ran and tested locally
- [x] Ticket reference is mentioned in linked commits (internal only)
- [ ] ~Breaking change is mentioned in relevant commits~

### CODE REVIEW

File: packages/manager/apps/hycu/public/translations/hycu/dashboard/Messages_fr_FR.json [hunk 1]
```diff
@@ -21,5 +21,18 @@
...
+  "hycu_dashboard_license_activate_heading": "Activer la licence",
+  "hycu_dashboard_license_activate_description": "Veuillez transférer votre fichier de demande de licence. Ce fichier est à télécharger depuis votre controller HYCU une fois votre installation réalisée",
+  "hycu_dashboard_license_more_information_link": "(plus de détails ici)",
+  "hycu_dashboard_activation_license_success_message": "La demande d'activation de votre licence a bien été prise en compte. Votre fichier de licence sera bientôt disponible sur cette interface et vous sera également envoyé par e-mail.",
+  "hycu_dashboard_activation_license_error_message": "Une erreur est survenue lors de la soumission de votre license.",
+  "hycu_dashboard_license_upload": "Upload",
+  "hycu_dashboard_upload_cancel": "Annuler",
+  "hycu_dashboard_upload_confirm": "Activer",
+  "hycu_dashboard_upload_valid_file": "Merci d'upload un fichier valide",
+  "hycu_dashboard_upload_license_required": "Veuillez upload une license",
```
Comment (darsene) [inline packages/manager/apps/hycu/public/translations/hycu/dashboard/Messages_fr_FR.json:34]: Could you use only french words

File: packages/manager/apps/hycu/public/translations/hycu/dashboard/Messages_fr_FR.json [hunk 2]
```diff
@@ -21,5 +21,18 @@
...
+  "hycu_dashboard_drag_and_drop_attachment": "Glissez-Déposer une pièce jointe",
+  "hycu_dashboard_accepted_formats": "Format acceptés: .dat",
```
Comment (darsene) [inline packages/manager/apps/hycu/public/translations/hycu/dashboard/Messages_fr_FR.json:36]:
```suggestion
  "hycu_dashboard_accepted_formats": "Format accepté: .dat",
```

Comment (pauldkn) [inline packages/manager/apps/hycu/public/translations/hycu/dashboard/Messages_fr_FR.json:36]: **typo** Format**s**

File: packages/manager/apps/hycu/src/hooks/api/license.ts [hunk 3]
```diff
@@ -1,15 +1,62 @@
...
 ) => {
   return useQuery({
-    queryKey: ['license/hycu', 'get', serviceName],
+    queryKey: getlicenseHycuQueryKey(serviceName),
     queryFn: () => getlicenseHycuService({ serviceName }),
     ...(options ?? {}),
   });
 };
+
+// https://eu.api.ovh.com/console/?section=%2Flicense%2Fhycu&branch=v1#post-/license/hycu/-serviceName-/activate
```
Comment (darsene) [inline packages/manager/apps/hycu/src/hooks/api/license.ts:31]:
```suggestion
```

File: packages/manager/apps/hycu/src/data/api/hycu.ts [hunk 4]
```diff
@@ -7,12 +7,21 @@ export type GetlicenseHycuListParams = {
   iamTags: string;
 };
 
-export const getlicenseHycuListQueryKey = ['get/license/hycu'];
+export const getlicenseHycuListQueryKey = () => ['license/hycu', 'get', 'list'];
+export const getlicenseHycuQueryKey = (serviceName: string) => [
```
Comment (pauldkn) [inline packages/manager/apps/hycu/src/data/api/hycu.ts:11]: **camelCase** `getLicense`

**Review by pauldkn — APPROVED** [review]

**Review by pauldkn — APPROVED** [review]

**Review by darsene — APPROVED** [review]

**Pull request merged**